common/src/main/java/com/wynntils/features/screens/CosmeticsPreviewFeature.java

```diff
@@ -17,7 +17,7 @@
 import net.minecraft.client.gui.screens.inventory.InventoryScreen;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
-@ConfigCategory(Category.UI)
+@ConfigCategory(Category.SCREENS)
 public class CosmeticsPreviewFeature extends Feature {
```

Comment:
As I said, this is just wrong. The preview is rendered on a Wynncraft inventory menu. It is not a "Screen", so it is not only not helpful, it is also not correct.